capten/agent/pkg/agent/plugin_tekton_apis.go

```diff
@@ -3,16 +3,61 @@ package agent
 import (
 	"context"
 	"fmt"
+	"os"
 
+	"github.com/intelops/go-common/credentials"
+	"github.com/kube-tarian/kad/capten/agent/pkg/model"
 	"github.com/kube-tarian/kad/capten/agent/pkg/pb/captenpluginspb"
 )
 
+const (
+	tekton string = "tekton"
+)
+
 func (a *Agent) RegisterTektonProject(ctx context.Context, request *captenpluginspb.RegisterTektonProjectRequest) (
 	*captenpluginspb.RegisterTektonProjectResponse, error) {
+	// get the corressponding git url from the DB.
+	projectUrl := os.Getenv("ProjectURL")
+	accessToken := os.Getenv("accessToken")
+	regTekton := &model.RegisterTekton{Id: request.Id, ProjectUrl: projectUrl, Status: "in-progress"}
```

Comment:
projectUrl := os.Getenv("ProjectURL")
	accessToken := os.Getenv("accessToken")

write a wrapper method to get instead of ENV